features/old/accounts/accounts.feature

```diff
@@ -61,34 +62,37 @@ Feature: Account management
     And I follow "Settings"
     Then I should not see "API Key"
 
-  @wip @javascript
+  @javascript
   Scenario: For admins the account overview is a page to change account details
     And current domain is the admin domain of provider "foo.3scale.localhost"
     When I log in as provider "foo.3scale.localhost"
       And I go to the provider account page
-    Then I should see the page to change account details
+    Then I should see "Account Details"
 
-  @wip
+  @javascript
   Scenario: Buyer Admins can edit Customers Type on account
-    Given provider "foo.3scale.localhost" has the following buyers with states:
-        | buyer | state    |
-        | buyer | approved |
-    When I log in as "buyer" on foo.3scale.localhost
-      And I follow "Settings"
-    Then I should be able to edit the value of the customers type field
+    And current domain is the admin domain of provider "foo.3scale.localhost"
+    When I log in as provider "foo.3scale.localhost"
+    Given a buyer "bob" signed up to provider "foo.3scale.localhost"
+
+    When I go to the buyer account page for "bob"
+    And I go to the fields definitions index page
+    And I follow "Create"
+    And create a new custom filed definition
 
+    When I log in as "buyer" on "foo.3scale.localhost"
+    And I follow "Settings"
+
+    And I follow "Edit Account"
     When I change the value of the customers type field to "Consumers"
-      And I fill in the obligatory fields for accounts
-      And I press the button to update account
-    Then I should see the page to change account details
-      And I should see the value of the customers type field is "Consumers"
+    Then I should see "Consumers"
 
-  @wip @javascript
+  @javascript
   Scenario: Provider Admins cannot edit Profiles fields on account
     And current domain is the admin domain of provider "foo.3scale.localhost"
     When I log in as provider "foo.3scale.localhost"
-      And I go to the provider account page
-    Then I should not be able to edit the value of the customers type field
+    And I go to the provider account page
+    Then I should not see "customers type"
```

Comment:
This is the same case as above, this cucumber was created to test a form in the Account Settings screen, but that screen changed and is not a form anymore, so the test doesn't make sense currently. I'm not sure how to proceed here. I'd say the best solution is to just remove this cucumber. Please ask @josemigallas and others about this.

Reply:
umm yes as the test doesn't make sense currently I can remove it